receiver/statsdreceiver/protocol/metric_translator.go

```diff
@@ -91,3 +91,34 @@ func buildSummaryMetric(summaryMetric summaryMetric) pdata.InstrumentationLibrar
 	return ilm
 
 }
+
+func (s statsDMetric) counterValue() int64 {
+	x := s.asFloat
+	// Note statds counters are always represented as integers.
+	// Since there is no statsd specification, it's difficult to
```

Comment:
>  Since there is no statsd specification, it's difficult to argue against this statement. 

Not sure how to interpret this statement. Is it something like "No known statsd implementations allow integer counters so we make sure we always export as integer"?

Reply:
I guess I'm making inside jokes with myself. I argued that this code should just use floating point counters to avoid rounding, but the implementor was copying behavior from CloudWatch and this was what he wanted. Lacking any specification, it was difficult to argue against. I still think it should change, but I don't have a leg to stand on.